app/helpers/iso19139_helper.rb

```diff
@@ -0,0 +1,45 @@
+module Iso19139Helper
+  # TODO: Migrate this code into an XSLT?
+  # TODO: Need to support multivalued fields
+  def extract_iso19139_metadata(doc)
+    ns = {
```

Comment:
Might `ns` here be restructured within the scope of the Module rather than the `#extract_iso19139_metadata` Method?

```
module Iso19139Helper
  NS = {
  [...]
  }

  def extract_iso19139_metadata(doc)
[...]
```

In this manner, I could (under very rare, but still existent cases) retrieve these values (should I need to parse XML Documents outside of the scope of this Module).

That being stated, again, this would be quite rare (most users parsing ISO19139 metadata are likely to simply include the namespaces within their own code base), so perhaps this is not of significant concern.